librz/core/cmd_descs/cmd_debug.yaml

```diff
@@ -4,4 +4,47 @@ commands:
   - name: dc
     cname: debug_continue_oldhandler
     summary: Continue execution
-    type: RZ_CMD_DESC_TYPE_OLDINPUT
\ No newline at end of file
+    type: RZ_CMD_DESC_TYPE_OLDINPUT
+  - name: ds
```

Comment:
this should probably be of type `RZ_CMD_DESC_TYPE_OLDINPUT`, otherwise you will lose access to all others `ds` commands.

Question: have you try running your branch and checking if the `ds`/`dsu` commands you have replaced still work well? I suggest that when you do such kind of conversion to newshell, you first spend some time understanding how the commands behave before your PR and then check that they still make sense after that.

For example, in this case if you do `ds?` you'd notice that a lot of commands are now missing because this is not OLDINPUT type. Also, if you check `dsuo` you'd see that it is not working as implemented right now. It is important that you test, at least briefly, manually your PRs when doing these kind of changes, to ensure functionalities are still maintained. Unfortunately, automatic tests do not cover everything, so we should pay attention